spec/classes/proxy_spec.rb

```diff
@@ -5,342 +5,332 @@
+          end
+
+          it { is_expected.to contain_class('zabbix::repo').with_zabbix_version('3.0') }
+          it { is_expected.to contain_package('zabbix-proxy-pgsql').with_require('Class[Zabbix::Repo]') }
+          it { is_expected.to contain_yumrepo('zabbix-nonsupported') }
+          it { is_expected.to contain_yumrepo('zabbix') }
+        end
+
+        describe 'when manage_repo is true and zabbix version is 2.4' do
+          let :params do
+            {
+              manage_repo: true,
+              zabbix_version: '2.4'
+            }
+          end
+
+          it { is_expected.to contain_class('zabbix::repo').with_zabbix_version('2.4') }
+          it { is_expected.to contain_package('zabbix-proxy-pgsql').with_require('Class[Zabbix::Repo]') }
+          it { is_expected.to contain_package('zabbix-proxy').with_ensure('present') }
+        end
+
+        describe 'with enabled selinux' do
+          let :facts do
+            super().merge(selinux_config_mode: 'enforcing')
+          end
+
+          it { is_expected.to contain_selboolean('zabbix_can_network').with('value' => 'on', 'persistent' => true) }
+        end
+
+        describe 'when database_type is postgresql' do
+          let :params do
+            {
+              database_type: 'postgresql'
+            }
+          end
+
+          it { is_expected.to contain_package('zabbix-proxy-pgsql').with_ensure('present') }
+          it { is_expected.to contain_package('zabbix-proxy-pgsql').with_name('zabbix-proxy-pgsql') }
+          it { is_expected.to contain_service('zabbix-proxy').with_require(['Package[zabbix-proxy-pgsql]', 'File[/etc/zabbix/zabbix_proxy.conf.d]', 'File[/etc/zabbix/zabbix_proxy.conf]']) }
+          it { is_expected.to contain_file('/etc/zabbix/zabbix_proxy.conf').with_require('Package[zabbix-proxy-pgsql]') }
+        end
+
+        describe 'when database_type is mysql' do
+          let :params do
+            {
+              database_type: 'mysql'
+            }
+          end
+
+          it { is_expected.to contain_package('zabbix-proxy-mysql').with_ensure('present') }
+          it { is_expected.to contain_package('zabbix-proxy-mysql').with_name('zabbix-proxy-mysql') }
+          it { is_expected.to contain_service('zabbix-proxy').with_require(['Package[zabbix-proxy-mysql]', 'File[/etc/zabbix/zabbix_proxy.conf.d]', 'File[/etc/zabbix/zabbix_proxy.conf]']) }
+          it { is_expected.to contain_file('/etc/zabbix/zabbix_proxy.conf').with_require('Package[zabbix-proxy-mysql]') }
+        end
+
+        describe 'when manage_resources is true' do
+          let :params do
+            {
+              manage_resources: true,
+              listenip: '192.168.1.1'
+            }
+          end
+
+          it { is_expected.to contain_class('zabbix::resources::proxy') }
+          it { is_expected.to contain_zabbix__userparameters('Zabbix_Proxy') }
+          it { is_expected.to contain_zabbix__resources__userparameters('foo_Zabbix_Proxy') }
```

Comment:
@bastelfreak Where did `foo_Zabbix_Proxy` come from?  Not sure why the tests on this PR all passed, but they don't anymore because of this.